requirements-base-lock.txt

```diff
@@ -0,0 +1,14 @@
+#
```

Comment:
If we install every lockfile at once do we have anything ensuring we don't have conflicts there? i.e. if we update one lockfile but forget to do the others, will we know before we merge?

Reply:
Hm that's a good point. At a minimum we should rework [regenerate-lock-files](https://github.com/aws/aws-cli/blob/v2/scripts/regenerate-lock-files) or make use of it to generate all of these at once.

This also raises another concern where we make a dependency in `pyproject.toml` more restrictive but forget to regenerate `requirements-docs-lock.txt` (which takes `pyproject.toml` as input).